.drone.star

```diff
@@ -145,7 +145,7 @@ config = {
         "earlyFail": True,
     },
     "e2eTests": {
-        "skip": False,
+        "skip": True,
```

Comment:
why do you need to skip the e2e tests?

Reply:
We split the change quota permission. There are now two. One for changing personal space quota and one for changing project space quota.

Since web is checking for the wrong permission e2e tests will fail when trying to change project space quota. Other way around web cannot get a green pipeline without ocis changes. 

We will reenable e2e tests as soon as web is bumped